app/services/feature_flag.rb

```diff
@@ -1,6 +1,6 @@
 module FeatureFlag
   class << self
-    delegate :disable, :enable, :enabled?, :exist?, to: Flipper
+    delegate :disable, :enable, :enabled?, :exist?, :remove, to: Flipper
```

Comment:
need `:remove` for the new data update script. We should also make a habit of adding/removing and controlling feature flags programmatically and not via the web UI if possible, to ensure all Forems are aligned